infra/examples-dev/aws-all/variables.tf

```
@@ -286,3 +286,14 @@ variable "todos_as_local_files" {
   description = "whether to render TODOs as flat files"
   default     = true
 }
+
+locals {
+  # tflint-ignore: terraform_unused_declarations
+  validate_salesforce_domain         = (var.salesforce_domain == null || var.salesforce_domain == "" || can(regex(":|\\/", try(var.salesforce_domain, "")))) && contains(var.enabled_connectors, "salesforce")
+  validate_salesforce_domain_message = "The salesforce_domain var should be populated and to be with only the domain (without protocol or query paths) itself if enabled."
```

I find this message a bit weird
```suggestion
  validate_salesforce_domain_message = "salesforce_domain var must be populated with just the domain (no protocol or query paths)"
```
if appears if because the connector is enabled